Sources/Remote Logging/Crash Logging/CrashLogging.swift

```diff
@@ -168,54 +171,24 @@ public extension CrashLogging {
     }
 
     /// Sends an `Event` to Sentry and triggers a callback on completion
-    func logErrorImmediately(_ error: Error, userInfo: [String: Any]? = nil, level: SentryLevel = .error, callback: @escaping (Result<Bool, Error>) -> Void) throws {
+    func logErrorImmediately(_ error: Error, userInfo: [String: Any]? = nil, level: SentryLevel = .error, callback: @escaping () -> Void) throws {
```

Comment:
Surprisingly, I'm not getting any errors or warnings from the compiler because of these methods still having `throws` despite there not being any genuine `try` call in the chain.

If we commit to `flush`, we should remove `throws` as well.